content/procedures/nominate-new-compiler-member.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Nominate new compiler team contributor or member
+type: docs
+---
+
+# Nominate new compiler team contributor or member
+
+This documents the process for nominating, inviting and onboarding new compiler team members or compiler team contributors.
+
+**Note:** [RFC 2689] is the authoritative reference on this topic.
+
+## Nominating new compiler team contributors
+
+1. A team member sends an email to `compiler-private@rust-lang.org` with the name/username of the individual being nominated and a brief explanation of why they are being nominated.
```

Comment:
We've discussed since retiring the mailing lists or using them less, so maybe we can change this to posting in our private Zulip stream?